### ISSUE TITLE
Modules empty blank page 

### ISSUE THREAD

**Issue by valedina** [formtools/core#580]
Hi Ben,
I've been trying for 3 days to upgrade FT from 2.2.6 (with all extensions updated) to 3.0.15 version through the website upgrade script, without success. I always receive the message "server too busy...". My FT was updated as its extensions and the System Check module confirmed that the system could run FT 3.0.

I have accordingly downloaded manually the 3.0 core and the extensions (versions as listed in the Upgrade script) and uploaded them in the previous FT folder (please note I have uploaded also the "install" folder). In the login page I didn't see the "You've been upgrade to ..." message, even if the version showed in the top right of the screen is 3.0.15. All the forms work fine, also "account" and "settings" pages, and the "pages" module. Unfortunately when I click on "Modules" on the side menu, an empty page is shown and I cannot manage the modules installed. I also removed the "core_fields_type" module from [form tools root]/modules/core_field_types/ folder, but the problem still persists. 

Could you please help me

Thanks!

**Comment by benkeen** [comment]
Hey @valedina, sorry you're having trouble - and I'm so sorry about the upgrade script not working properly, that thing's been the bane of my existence for longer than I'd care to admit. 

Hmm, removing the core field types module is usually the cause of that issue. Try this:L 

1. Add the following to your `/global/config.php` file:

```php
$g_default_error_reporting = 2047;
```

Then refresh the modules page. Do you see any errors output to the screen? 

2. if that doesn't provide any useful information, check your PHP error logs on your server. This can take some hunting and may require you to bug your hosting provider, but that file can give you some important clues about what error is actually happening to help us out. 

Ben

**Comment by valedina** [comment]
thank you for your prompt reply! 
this is the output error message  

Fatal error: Call to a member function getModuleName() on boolean in /web/htdocs/[...]/admin/modules/index.php on line 54

I also followed your suggestion about the theme cache

thank you

**Comment by jgold723** [comment]
I'm having the same problem. I upgraded from 3.0.8 to 3.0.15. The upgrade seems to have gone fine, except I cannot view the modules page - -it's blank. I've tried:
--Reuploading the files;
--Adding the error reporting (the page is still blank with no errors reported;
--Checked my modules folder and there is no core_field_types directory ;
--Cleared the cache for my theme.

I checked Valedina's link to issue 232, but that seems to be the core field types?
